src/common/components/hosts/SkipFormattingCheckbox.tsx

```diff
@@ -35,6 +35,23 @@ export const isDiskFormattable = (host: Host, diskId: string | undefined) => {
   );
 };
 
+export const isSkipFormattingChecked = (host: Host, diskId: string | undefined) => {
+  return isDiskFormattable(host, diskId) && !isDiskSkipFormatting(host, diskId);
+};
+
+export const isSkipFormattingDisabled = (
+  host: Host,
+  diskId: string | undefined,
+  installationDiskId: string | undefined,
+) => {
+  return (
+    !isDiskFormattable(host, diskId) ||
+    (isDiskFormattable(host, diskId) &&
```

Comment:
`isDiskFormattable(host, diskId)` is unnecessary in the second part because the first part is false, so is must be formattable